reflex/reflex.py

```diff
@@ -629,12 +647,13 @@ def deployv2(
     vmtype: Optional[str] = typer.Option(
         None,
         "--vmtype",
-        help="Vm type id. Run `reflex apps vmtypes` to get options.",
+        help="Vm type id. Run reflex apps vmtypes list to get options.",
```

Comment:
should leave the backticks
```suggestion
        help="Vm type id. Run `reflex apps vmtypes list` to get options.",
```